File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        
	
	int a_pointer = 0;
	int b_pointer = nums.length-1;
	
	while(a_pointer<=b_pointer) {
		
		int sum = nums[a_pointer]+nums[b_pointer];
		
		
		if (sum < target) {
			a_pointer += 1;
			
		} else if (sum > target) {
			b_pointer -= 1;
			
		} else {
			
			return new int[] {a_pointer+1, b_pointer +1};
			
		}
	}
	
	
	return new int[] {a_pointer+1, b_pointer +1};
	
    }
}

File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers, target):
        i = 0;
        j = len(numbers)-1

        while(i<j):
            num_i = numbers[i]
            num_j = numbers[j]
            sum = num_i + num_j
            if(sum == target):
                return [i+1,j+1]
            else:
                if(sum>target):
                    j -= 1
                else:
                    i += 1

